src/testing/useful-crates.md

```diff
@@ -1,13 +0,0 @@
----
-minutes: 3
----
-
-# Useful Crates
-
-Rust comes with only basic support for writing tests.
-
-Here are some additional crates which we recommend for writing tests:
-
-* [googletest](https://docs.rs/googletest): Comprehensive test assertion library in the tradition of GoogleTest for C++.
-* [proptest](https://docs.rs/proptest): Property-based testing for Rust.
-* [rstest](https://docs.rs/rstest): Support for fixtures and parameterised tests.
```

Comment:
Do we no longer recommend proptest or rstest?

Reply:
I think we (meaning Android) never really recommended them. Personally, I've never used them and I've also not seem them being used in the libraries I've looked at.